arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,214 @@
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_GetServerPolicy",
+                "description": "Returns an Azure SQL Server security alert policy.",
+                "parameters": [
+                    {
+                      "name": "resourceGroupName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Resource Group to which the server belongs."
+                    },
+                    {
+                      "name": "serverName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Azure SQL Server."
+                    }, 
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+  "definitions": {
+    "ServerSecurityAlertPolicyProperties": {
+      "properties": {
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The state of the Azure SQL Security alert policy (New, Enabled or Disabled)."
+        },
+        "disabledAlerts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The disable alerts of the Azure SQL Security alert policy."
+        },
+        "emailAddresses": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The list of email addresses to send the alert."
+        },
+        "emailAccountAdmins": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Value that indicated whether to email the azure account admins."
```

Comment:
Shouldnt this be a Boolean? What kind of value do you expect the user to provide. If there is a specific set of values that the server expects then should this be an enum ?